db/seeds.rb

```diff
@@ -5,3 +5,20 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+user = User.create(email: "user@gmail.com", password: "password")
+
+Dream.create(user: user, date: Date.today, text: "My dream has two main sections and is very long. It begins at a different version of our older home on Stadcor Street in Wavell Heights. However, our house is built directly over the ground (instead of up on stumps) and has a dirt floor, which does not trigger the fact that I am dreaming. I notice a large garden in our living room growing directly from the \"floor\". They are large-leafed vine-like plants; possibly cantaloupes. Over time, however, I want to make our house more like a modern home and have our garden outside. It seems somewhat frustrating to me though I do notice sheets of vinyl and carpet over some areas of the dirt floor in different rooms, but only covering about half the area. This combination of vinyl or carpet over a dirt floor seems more aggravating than just having a dirt floor throughout.
```

Comment:
Line is too long. [873/80]<br>Do not use `Date.today` without zone. Use `Time.zone.today` instead.